app/src/main/java/com/kickstarter/ui/activities/ThanksActivity.kt

```diff
@@ -0,0 +1,130 @@
+package com.kickstarter.ui.activities
+
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Bundle
+import android.util.Pair
+import android.view.View
+import androidx.appcompat.app.AlertDialog
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.kickstarter.R
+import com.kickstarter.databinding.ThanksLayoutBinding
+import com.kickstarter.libs.BaseActivity
+import com.kickstarter.libs.KSString
+import com.kickstarter.libs.RefTag
+import com.kickstarter.libs.qualifiers.RequiresActivityViewModel
+import com.kickstarter.libs.rx.transformers.Transformers
+import com.kickstarter.libs.utils.ViewUtils
+import com.kickstarter.models.Project
+import com.kickstarter.services.DiscoveryParams
+import com.kickstarter.ui.IntentKey
+import com.kickstarter.ui.adapters.ThanksAdapter
+import com.kickstarter.viewmodels.ThanksViewModel
+import java.util.concurrent.TimeUnit
+
+@RequiresActivityViewModel(ThanksViewModel.ViewModel::class)
+class ThanksActivity : BaseActivity<ThanksViewModel.ViewModel>() {
+    private lateinit var ksString: KSString
+    private lateinit var binding: ThanksLayoutBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ThanksLayoutBinding.inflate(layoutInflater)
```

Comment:
nit: If we could separate blocks of code with a whitespace similarly to the java code, that would help with readability.